app/javascript/hyacinth_ui_v1/graphql/users.js

```diff
@@ -1,18 +1,19 @@
 import { gql } from 'apollo-boost';
 
-export const GetUsersQuery = gql`
-  query {
+export const getUsersQuery = gql`
+  query Users {
```

Comment:
I thought this query was working - does adding the `Users` tag break the regular user listing?

Reply:
It was working fine, but I added the query name here just because of all other named queries I saw in `projects.js`.  I'm actually not sure why all of the other queries are named, but I did it for consistency.  Maybe @cgalarza has thoughts about this.